purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -2,12 +2,23 @@
 
 from collections import defaultdict
 
-from odoo import _, models
+from odoo import _, api, fields, models
 
 
 class PurchaseRequisition(models.Model):
     _inherit = "purchase.requisition"
 
+    rfq_partner_ids = fields.Many2many(
+        comodel_name="res.partner", compute="_compute_rfq_partners"
+    )
+
+    def _compute_rfq_partners(self):
+        rfqs = self.env["purchase.order"].search(
+            [("requisition_id", "=", self.id), ("state", "!=", "cancel")]
+        )
+        for rec in self:
+            rec.partner_rfq_ids = rfqs.mapped("partner_id").ids
+
```

Comment:
If I'm not wrong, you're no longer using this field, right? Then, you can remove it:

```suggestion
```